site/en/guides/border-router/build-native.md

```diff
@@ -27,12 +27,14 @@ OTBR has two scripts that bootstrap and set up the Thread Border Router:
 $ cd ot-br-posix
 $ ./script/bootstrap
 $ INFRA_IF_NAME=wlan0 ./script/setup
+$ ./script/server
 ```
 
 OTBR works on both a Thread interface and infrastructure network interface (e.g. Wi-Fi/Ethernet) which is specified with `INFRA_IF_NAME`. The Thread interface is created by OTBR itself and named `wpan0` by default and the infrastructure interface has a default value of `wlan0` if `INFRA_IF_NAME` is not explicitly specified. If your Raspberry Pi is connected by an Ethernet cable, specify the Ethernet interface name (e.g. `eth0`):
 
 ```
 $ INFRA_IF_NAME=eth0 ./script/setup
+$ ./script/server
 ```
 
```

Comment:
How about adding a separate step with note as below, other than duplicate it in two places?

> If you want to start otbr-agent service without reboot, please run below script

```
$./script/server 
```